microsoft-edge/devtools-guide-chromium/experimental-features/focus-mode.md

```diff
@@ -148,27 +153,27 @@ Currently, the following tools cannot be unpinned from the **Activity Bar**:
 <!-- ====================================================================== -->
 ## Open tools from Quick View
 
-Use the **Quick View** list to open a second tool next to the one already selected from the **Activity Bar**.
+Use the **Quick View** to open a second tool next to the one already selected from the **Activity Bar**:
```

Comment:
fts Findings about current usage fwiw:

![image](https://user-images.githubusercontent.com/45407486/223806777-33d40a47-0cfe-47ee-87fc-bc43a7507f36.png)

@captainbrosset  - for consideration: how good are these existing phrases?  this only shows hits that start with "the", like above screenshot.

the quick view list   4x
the quick view panel 1x
the quick view options 1x
the quick view pane 2x